Modules/HR/Database/Seeders/SettingsTableSeeder.php

```diff
@@ -59,6 +59,17 @@ public function run()
             'setting_key' => 'applicant_verification_subject',
             'setting_value' => 'Confirm your email address',
         ]);
+        Setting::updateOrCreate([
+            'module' => 'hr',
+            'setting_key' => 'Follow_up_email_for_scheduling_interview_subject',
```

Comment:
key should be in lowercase
```suggestion
            'setting_key' => 'follow_up_email_for_scheduling_interview_subject',
```

Reply:
Done!